datafusion/core/Cargo.toml

```diff
@@ -40,7 +40,7 @@ nested_expressions = ["datafusion-functions-nested"]
 # This feature is deprecated. Use the `nested_expressions` feature instead.
 array_expressions = ["nested_expressions"]
 # Used to enable the avro format
-avro = ["apache-avro", "num-traits", "datafusion-common/avro"]
+avro = ["apache-avro", "num-traits", "datafusion-common/avro", "datafusion-datasource/avro"]
```

Comment:
I wonder if we can remove the avro stuff from datafusion-common 🤔

Reply:
I think the next PR will try and do that, but some will always stay even if just for stuff like `AvroError`/`ParquetError`